demisto_sdk/commands/prepare_content/preparers/incident_to_alert.py

```diff
@@ -0,0 +1,73 @@
+import re
+
+from demisto_sdk.commands.common.constants import MarketplaceVersions
+
+
+def prepare_descriptions_and_names(data: dict, marketplace: MarketplaceVersions) -> dict:
+    # Replace incidents to alerts only for XSIAM
+    replace_incident_to_alert = marketplace == MarketplaceVersions.MarketplaceV2
+
+    for k, v in data.get("tasks", {}).items():
+        if description := v.get("task", {}).get("description", ""):
+            if replace_incident_to_alert:
+                description = name_or_description_incident_to_alert(description)
+
+            remove_wrapper_from_incident(data, "description", description, k)
```

Comment:
this pattern is repeated 4 times, can we make it into a function? Have it return a value, then assign it back to wherever it fits.